docs/reference/authentication.md

```diff
@@ -27,6 +27,12 @@ Web UI at Administration / Users to create users. Users have an access key
 for logging into the Web UI or authenticating programmatic requests to the API
 Server or the S3 Gateway.
 
+#### Remote Authenticator Service 
+
+LakeFS server supports external authentication, the feature can be configured by providing an HTTP endpoint to an external authentication service. This integration can be especially useful if you already have an existing authentication system in place, as it allows you to reuse that system instead of maintaining a new one.
```

Comment:
@ortz how much guidance do we plan for explaining external authentication?

Reply:
let's see how many requests we got for this and we'll invest efforts accordingly.

Reply:
👍